src/main/java/ua/gorshkov/hw19/DBPersonDAO.java

```diff
@@ -0,0 +1,82 @@
+package ua.gorshkov.hw19;
+import org.hibernate.Session;
+import java.util.List;
+
+public class DBPersonDAO implements DAO<Person> {
```

Comment:
Relates to all methods. In try you should open transaction, do smth and in catch in case of exception do rollback:

For intance,
```
try {
            session.beginTransaction();
            session.save(obj);
            session.getTransaction().commit();
        } catch (Exception e) {
            session.getTransaction().rollback();
            throw new RuntimeException("Error while committing the transaction", e);
        }
```